packages/flutter/lib/src/widgets/modal_barrier.dart

```diff
@@ -14,6 +14,97 @@ import 'gesture_detector.dart';
 import 'navigator.dart';
 import 'transitions.dart';
 
+/// A widget that modifies the size of the [SemanticsNode.rect] (focus) of its
+/// child widget. It clips the focus in possibily four directions based on the
```

Comment:
the first paragraph of the doc string should be one sentence. Maybe put the second sentence to its own paragraph

Reply:
Thanks! Done.